docs/kp_secret_list.md

```diff
@@ -1,13 +1,15 @@
 ## kp secret list
 
-List secrets
+List secrets for a service account
```

Comment:
```suggestion
List secrets attached to a service account?
```

Reply:
How about `List secrets linked to a service account`?

Reply:
Attached. I'm good with attached.